src/routes/auth/signin.e2e.test.js

```diff
@@ -0,0 +1,135 @@
+const { expect } = require('chai')
+const { fixtures } = require('../../../test/utils')
+
+describe('POST /auth/signin', async() => {
+	let user
+	let validPassword 
+	let signinInfo
+	before(async ()=> {
+		validPassword = 'validPassword123$'
+		user = await fixtures.insertUser({
+			password : validPassword
+		})    
+		signinInfo = {
+			email : user.email
+		}
+	})
+	describe('Failure', async() => {
+		it(`should return a 400 error if request body is absent`, async() => {
+			const expectedError = {
+				"error": {
+					"message": 
+				"email is required, password is required"
+				},
+				"status": "failed"
+			}
+
+			return fixtures.api()
+				.post('/api/v1/auth/signin')
```

Comment:
lets start doing this, kindly create a global variable at the top which says `endpointRoute` or something better and assign the value of `/api/v1`auth/signin` to it. 

so we only have to just change it at one place in case the endpoint change for some reason

Reply:
yeah, that sounds really cool